default/config/FpConfig.h

```diff
@@ -323,7 +299,7 @@ typedef enum {
 #endif
 
 #ifndef FW_RAW_TIME_HANDLE_MAX_SIZE
-#define FW_RAW_TIME_HANDLE_MAX_SIZE 32  //!< Maximum size of a handle for OS::RawTime objects
+#define FW_RAW_TIME_HANDLE_MAX_SIZE 56  //!< Maximum size of a handle for OS::RawTime objects
```

Comment:
Do we know why this went up?  I would have expected this to be almost a no-op in terms of size, so I am surprised that we had to add on 24 more bytes.

Reply:
Possible Does the FPP enum type store a bunch of members that aren't serialized or otherwise take a bunch of space over a enum integer?